scripts/entrypoint.sh

```diff
@@ -12,7 +12,7 @@ TITAN_IDS=titan_ids
 
 export JAVA_OPTIONS=${JAVA_OPTIONS:- -Xms512m -Xmx1400m}
 
-export JAVA_OPTIONS="$JAVA_OPTIONS -javaagent:/opt/dynamodb/$SERVER_DIR/lib/jamm-0.3.0.jar"
+export JAVA_OPTIONS="$JAVA_OPTIONS -javaagent:/metrics/jmx_prometheus_javaagent.jar=5000:/metrics/config.yaml"
```

Comment:
you've cross verified that this works fine as before?

Reply:
@yzainee I had verified with my ephemeral openshift cluster. Don't have access in PSI. Let me try with staging.